src/app/client/src/app/modules/shared/services/config/app.config.json

```diff
@@ -924,6 +924,7 @@
       "application/vnd.ekstep.content-collection"
     ],
     "objectType": "Content",
+    "allObjectType": ["Content", "QuestionSet"],
```

Comment:
please make sure this name is more proper like allowedObjectTypes

Reply:
Updated the name.